src/porepy/numerics/fv/biot.py

```diff
@@ -608,10 +611,344 @@ def compute_stress(self, g, u, data):
+        return solution_array
+
+    def discretize(self, g, data):
+        """ Discretize the displacement divergence term of the Biot equation.
+
+        Parameters:
+            g (pp.Grid): grid, or a subclass, with geometry fields computed.
+            data (dict): For entries, see above.
+
+        Raises:
+            NotImplementedError, the discretization should be performed using the
+            discretize method of the Biot class.
+        """
+        raise NotImplementedError(
+            """No discretize method implemented for the GradP
+                                  class. See the Biot class."""
+        )
+
+    def assemble_matrix_rhs(self, g, data):
+        """ Return the matrix and right-hand side for a discretization of the
+        displacement divergence term of the Biot equation.
+
+        Parameters:
+            g : grid, or a subclass, with geometry fields computed.
+            data: dictionary to store the data. For details on necessary keywords,
+                see method discretize()
+
+        Returns:
+            matrix: sparse csr (g.dim * g_num_cells, g.dim * g_num_cells) Discretization
+            matrix.
+            rhs: array (g.dim * g_num_cells) Right-hand side.
+        """
+        return self.assemble_matrix(g, data), self.assemble_rhs(g, data)
+
+    def assemble_matrix(self, g, data):
+        """ Return the matrix and right-hand side for a discretization of the
+        displacement divergence term of the Biot equation.
+
+        Parameters:
+            g (Grid): Computational grid, with geometry fields computed.
+            data (dictionary): With data stored.
+
+        Returns:
+            scipy.sparse.csr_matrix: System matrix of this discretization. The
+                size of the matrix will depend on the specific discretization.
+
+        Raises:
+            ValueError if the displacement divergence term has not already been
+            discretized.
+        """
+        matrix_dictionary = data[pp.DISCRETIZATION_MATRICES][self.keyword]
+        if not "div_d" in matrix_dictionary:
+            raise ValueError(
+                """DivD class requires a pre-computed discretization to be
+                             stored in the matrix dictionary."""
+            )
+        biot_alpha = data[pp.PARAMETERS][self.keyword]["biot_alpha"]
+        return matrix_dictionary["div_d"] * biot_alpha
+
+    def assemble_rhs(self, g, data):
+        """ Return the zero right-hand side for a discretization of the displacement
+        divergence term.
+
+        @Runar: Is it correct that this is zero.
+
+        Parameters:
+            g (Grid): Computational grid.
+            data (dictionary): With data stored.
+
+        Returns:
+            np.ndarray: Zero right hand side vector with representation of boundary
+                conditions.
+        """
+        return np.zeros(self.ndof(g))
+
+
+class BiotStabilization(pp.numerics.mixed_dim.VectorEllipticDiscretization):
```

Comment:
I guess it will be up to me to write the explanation of what on earth this term really is. I will return to it one day I feel particularly brave.